v23.1/connect-to-a-changefeed-kafka-sink-with-oauth-using-okta.md

```diff
@@ -0,0 +1,291 @@
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    vi server.properties
+    ~~~
+
+    Add the following to the end of the `server.properties` file:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~
+    sasl.enabled.mechanisms=OAUTHBEARER
+    sasl.mechanism.inter.broker.protocol=OAUTHBEARER
+    security.inter.broker.protocol=SASL_PLAINTEXT
+    listeners=PLAINTEXT://:9092,SASL_PLAINTEXT://:9093
+    advertised.listeners=PLAINTEXT://:9092,SASL_PLAINTEXT://:9093
+    auto.create.topics.enable=true
+
+    listener.name.sasl_plaintext.oauthbearer.sasl.login.callback.handler.class=br.com.jairsjunior.security.oauthbearer.OauthAuthenticateLoginCallbackHandler
+    listener.name.sasl_plaintext.oauthbearer.sasl.server.callback.handler.class=br.com.jairsjunior.security.oauthbearer.OauthAuthenticateValidatorCallbackHandler
+    ~~~
+
+    This defines:
+    - The SASL mechanism as `OAUTHBEARER`.
+    - The ports that Kafka brokers listen for client and inter-broker communication.
+    - The login callback handler that will receive the OAuth token.
+    - The server callback handler that will verify the token.
+
+1. Create a file in the `config` directory called `kafka_server_jaas.conf`:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    vi kafka_server_jaas.conf
+    ~~~
+
+    Add the following contents:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~
+    KafkaServer {
+    org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required;
+    };
+    ~~~
+
+    The [`OAuthBearerLoginModule`](https://kafka.apache.org/20/javadoc/org/apache/kafka/common/security/oauthbearer/OAuthBearerLoginModule.html) will ask the `OauthAuthenticateLoginCallbackHandler` (configured in the `server.properties` file) to return an `OAuthBearerToken`.
+
+## Step 4. Prepare Kafka OAuth libraries
+
+The SASL callback handlers added in [Step 3](#step-3-update-kafka-configuration) are part of a separate library that you will need to install.
+
+1. Clone the library:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    git clone https://github.com/jairsjunior/kafka-oauth.git ~/kafka-oauth
+    ~~~
+
+1. Install [Apache Maven](https://maven.apache.org/what-is-maven.html), a tool for building Java-based projects:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    sudo apt install maven
+    ~~~
+
+1. Move to the `kafka-oauth` directory: 
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cd ~/kafka-oauth
+    ~~~
+
+1. Run the following to compile and package the project:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    mvn package
+    ~~~
+
+## Step 5. Add OAuth environment variables
+
+In this step, you'll export the environment variables that contain your OAuth client credentials. You will need the following information from your Okta developer account:
+
+- Your Okta developer account URL, for example `dev-12345678.okta.com`
```

Comment:
Wondering if this should just be called "your Okta domain" https://developer.okta.com/docs/guides/find-your-domain/main/

Reply:
I think so, changing this